index.js

```diff
@@ -19,7 +19,7 @@ function fastifyRequestContext(fastify, opts, next) {
   fastify.addHook(hook, (req, res, done) => {
     const defaultStoreValues =
```

Comment:
can you also do a microoptimization and precalculate results of typeof check?

Reply:
Fixed ✔️